pkg/domain/infra/abi/play.go

```diff
@@ -235,6 +237,16 @@ func (ic *ContainerEngine) PlayKube(ctx context.Context, body io.Reader, options
 			}
 			notifyProxies = append(notifyProxies, proxies...)
 
+			pod, err := ic.Libpod.GetPod(r.Pods[0].ID)
+			if err != nil {
+				return nil, err
+			}
+
+			mountLabel, err = pod.MountLabel()
+			if err != nil {
+				return nil, err
+			}
+
```

Comment:
I don't think that can work correctly, here we loop over all content in the file.
I don't see any guarantee that a pod must be before the PersistentVolumeClaim. Right now it is just depends on the user, that seems wrong.

Why even change play kube? There is no test for it or any description why kube was changed? I understand the ` --security-opt` change but kube does not support it AFAIK.

Reply:
Kube yaml can support filetype labels (At least in Podman) Now.

I can change this to just pass in "", since it is arbitrary, but I am not sure that is correct in the general case.